### PR TITLE
Use double qoutes in lib/omniauth/strategies/qualtrics.rb

### PR DESCRIPTION
**Pull request by domschab23**
Per maintainer request https://github.com/atomicjolt/omniauth-qualtrics/pull/1#pullrequestreview-2222304496, he would like us to use `""` instead of `''` to be consistent with the rest of a project before merging our fix

### CODE REVIEW

File: lib/omniauth/strategies/qualtrics.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
-require 'omniauth-oauth2'
+require "omniauth-oauth2"
```
Comment (domschab23) [inline lib/omniauth/strategies/qualtrics.rb:1]: I know this goes against rubocop guidelines, but this is the request from repo maintainer

Comment (jeffbax) [reply lib/omniauth/strategies/qualtrics.rb:1]: when forking, you should stick to the rubocop of the project if there is one rather than ours

Comment (jeffbax) [reply lib/omniauth/strategies/qualtrics.rb:1]: and just generally make the diff as small as possible

Comment (smcabrera) [reply lib/omniauth/strategies/qualtrics.rb:1]: > I know this goes against rubocop guidelines, but this is the request from repo maintainer

Yeah it looks like it's consistent with their own [rubocop config](https://github.com/user-interviews/omniauth-qualtrics/blob/master/.rubocop.yml). 

I'm not sure why my tooling didn't just use pick up this config automatically back then, but thanks for fixing 👍

**Review by smcabrera — APPROVED** [review]

**Pull request merged**